### ISSUE TITLE
@stylexjs/eslint-plugin: "@stylexjs/sort-keys" severity invalid

### ISSUE THREAD

**Issue by olivierpascal** [facebook/stylex#414]
### Describe the issue

> [Error] .eslintrc.js: Configuration for rule "@stylexjs/sort-keys" is invalid: Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"warning"').

### Expected behavior

No error.

### Steps to reproduce

```js
// .eslintrc.js

// ...
  rules: {
    '@stylexjs/valid-styles': 'error',
    '@stylexjs/sort-keys': [
      'warning',
      {
        minKeys: 3,
        allowLineSeparatedGroups: true,
      },
    ],
  }
// ...
```

**Comment by nedjulius** [comment]
hey, it should be `warn` instead of `warning`. also fixed this in the #413

**Comment by nmn** [comment]
I just merged #413
